greek/put_socks_off/put-socks-off.md

```diff
@@ -2,5 +2,8 @@
 άραγε παρόμοια αισθητήρια όπως και εσύ; Βλέπει, ακούει, μυρίζει; 
 Ας δούμε λοιπόν πως αντιδράει στην μοχθηρή μυρωδιά μιας
 χιλιο-φορεμένης καλτσας...
-... ίσως το ζαχαρωτό να αρχίσει να δακρύζει. Τι θα μας έδειχνε αυτό;
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.